### PR TITLE
Fixed pretty much everything and corrected spelling

### PR DESCRIPTION
**Pull request by Kek5chen**
This project was far from running under WSL or (probably) linux for that matter. The line separators were all over the place from people committing from several platforms and there were some wrong or not working safety checks next to unused code and lots more.... this fixes most of the things

not even the readline library check was implemented properly. I added an automatic request to pull the readline library to the user the first time it needs to be installed to make it a bit easier.

### CODE REVIEW

**Comment by JCluzet** [comment]
Hey @Kek5chen 👋
Thank you very much for your modifications! 
It's very great, I actually made a lot of mistakes on the word "exercise", and everything else you've fixed is amazing.
Unfortunately I see that you deleted the VIP list on exam.cpp and made everyone a basic VIP.
I can't merge this change.

As soon as you put it back the way it was before, I'll do the merge with great pleasure :)

**Comment by Kek5chen** [comment]
oh, that was just for me. although I also have to say I'm not a fan of open source software having paid features. Just give me a minute and i'll make the change.

**Comment by Kek5chen** [comment]
there you go, everything back to normal 👍

File: .system/launch.sh [hunk 1]
```diff
@@ -140,7 +148,7 @@ rm -rf .system/readline_ok
 # echo -ne "Compilation of$BOLD$MANGENTA 42_EXAM v2.1 $RESET "
 # ===============================================
 
-g++ .system/exercise.cpp .system/main.cpp .system/menu.cpp .system/exam.cpp .system/utils.cpp .system/grade_request.cpp .system/data_persistence.cpp -lreadline -o .system/a.out >.system/.devmake.err 2>.system/.devmake.err &
+g++ .system/exercice.cpp .system/main.cpp .system/menu.cpp .system/exam.cpp .system/utils.cpp .system/grade_request.cpp .system/data_persistence.cpp -lreadline -o .system/a.out >.system/.devmake.err 2>.system/.devmake.err &
```
Comment (Kek5chen) [inline .system/launch.sh:151]: i changed the file name too. that's why that has also been changed

**Comment by JCluzet** [comment]
I totally understand.
For me it's not a paid feature, it's a way to offer people to donate to me, they set the price themselves. 
I just clarified my ReadMe that you can now be a VIP by doing a Sponsor **OR** by contributing.

So I can put you there directly, tell me your username so you can keep the version up to date with feature without having to make the change every time ;)

Thanks to you for all these changes and the work you have done, it's very great!

**Comment by Kek5chen** [comment]
> I just clarified my ReadMe that you can now be a VIP by doing a Sponsor OR *by contributing*.

That seems pretty reasonable.

My username is 'kx' on my main station and mac. Thanks 👍🏼

**Comment by JCluzet** [comment]
It's added :)

**Pull request merged**